src/style.css

```diff
@@ -2,0 +1,30 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0) {
```

Comment:
The selector `.stars--0 .stars__star:nth-child(-n + 0)` will never match any elements because `nth-child(-n + 0)` selects no elements. You can remove this rule if it's not intended to apply any styles.